waltz-ng/client/data-flow/components/data-flows-tabgroup/data-flows-tabgroup.html

```diff
@@ -125,9 +125,11 @@
                 <div ng-if="$ctrl.flowData.flows.length >0"
                      class="waltz-scroll-region"
                      style="height: 300px">
-                    <waltz-data-flow-table flows="$ctrl.filteredFlowData.flows">
```

Comment:
Suspect this is the only usage of 'waltz-data-flow-table'... perhaps we can remove it ?